Cython/Utility/AsyncGen.c

```diff
@@ -423,6 +423,36 @@ __Pyx_async_gen_unwrap_value(__pyx_PyAsyncGenObject *gen, PyObject *result)
     return result;
 }
 
+static PyObject *
+__Pyx_async_gen_unwrap_amvalue(__pyx_PyAsyncGenObject *gen, PyObject *result, __Pyx_PySendResult send_result)
```

Comment:
This is kind of duplicated though, since we still need the old one for the versions that call throw rather than send.